composer/src/main/kotlin/com/gojuno/composer/Apk.kt

```diff
@@ -0,0 +1,50 @@
+package com.gojuno.composer
+
+import com.gojuno.commander.android.aapt
+import com.gojuno.commander.os.Notification
+import com.gojuno.commander.os.process
+
+fun parseTestPackage(testApkPath: String): String =
+        process(
+                commandAndArgs = listOf(
+                        aapt, "dump", "badging", testApkPath
+                ),
+                unbufferedOutput = true
+        )
+                .ofType(Notification.Exit::class.java)
+                .map { (output) ->
+                    output
+                            .readText()
+                            .split(System.lineSeparator())
+                            // output format `package: name='$testPackage' versionCode='' versionName='' platformBuildVersionName='xxx'`
+                            .first { it.contains("package") }
+                            .split(" ")
+                            .first { it.startsWith("name=") }
+                            .split("'")[1]
+                }
+                .onErrorReturn { "" }
```

Comment:
Lets propagate error up or make a sealed class for result? Otherwise we're kinda using one possible state of string as error indicator